tests/pipelines/test_pipelines_image_segmentation.py

```diff
@@ -313,18 +321,17 @@ def test_threshold(self):
     @require_torch
     @slow
     def test_maskformer(self):
-        threshold = 0.999
+        threshold = 0.8
```

Comment:
Update needed since the pipeline now actually flows the information to the `feature_extractor`.